tool/password/src/main/java/com/github/mizool/tool/password/PasswordHashTool.java

```diff
@@ -39,12 +43,16 @@ public static void main(String[] args)
             }
             else
             {
-                PasswordHashTool passwordHashTool = new PasswordHashTool(parameters.getPassword());
-                String digest = passwordHashTool.getDigest();
-                if (digest != null)
-                {
-                    System.out.println("Digest: " + digest);
-                }
+                Iterable<PasswordHasher> instances = MetaInfServices.instances(PasswordHasher.class);
+                instances.forEach(hasher -> {
+                    PasswordHashTool passwordHashTool = new PasswordHashTool(hasher, parameters.getPassword());
+
+                    String digest = passwordHashTool.getDigest();
+                    if (digest != null)
```

Comment:
I have no idea why we added this null check here. If a hasher _did_ have trouble with a password, it should throw an exception. Having it return `null` and silently skip it is not the way to go. Remove the check and inline the `digest` variable.